packages/docs/.env.example

```diff
@@ -1 +1 @@
-VITE_PUBLIC_CLOUDINARY_CLOUD_NAME=
+VITE_CLOUDINARY_CLOUD_NAME=
```

Comment:
can you add the .env.example in playground too? since it needs to be separately configured